README.md

```diff
@@ -200,7 +200,7 @@ Shoes 4 is developed in a TDD style using RSpec. You should be writing and runni
 
 The simplest way to do this is with rake tasks.
 
-    $ rake spec
+    $ rake
```

Comment:
It still fails? I thought I'd fixed that.

Either way, I'll update this as my rule is ABB... "Always Be Bundling"